TruePath/AbsoluteDirectory.cs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 TruePath contributors <https://github.com/ForNeVeR/TruePath>
+//
+// SPDX-License-Identifier: MIT
+
+namespace TruePath;
+
+public static class AbsoluteDirectory
```

Comment:
Another general design note: let's mark all these methods as extensions on `AbsolutePath`.

There's a simple reason they weren't made extensions originally: it'd be awkward to define them as extensions on `string` (since .NET lacked a path type). In TruePath, we don't have this problem, so let's mark them as extensions.